src/telemetry/component-names.ts

```diff
@@ -55,3 +55,11 @@ export const AUTHENTICATION_ACTION = 'Authentication action';
 export const GET_ADAPTIVE_CARD_ACTION = 'Get adaptive card action';
 export const FETCH_PERMISSIONS_ACTION = 'Fetch permissions action';
 export const MONACO_EDITOR = 'Monaco editor display';
+export const ADD_RESOURCE_TO_COLLECTION = 'Add resource to collection action';
+export const REMOVE_RESOURCE_FROM_COLLECTION = 'Remove resource from collection action';
+export const SET_QUERY_ACTION = 'Resources set query action';
```

Comment:
Identifier should indicate that this is an action for resources to prevent misuse. It could be renamed to `RESOURCES_SET_QUERY_LISTITEM`